### PR TITLE
updated oc docs with bad feature warning

### PR DESCRIPTION
**Pull request by k-dominik**
ref: https://github.com/ilastik/ilastik/pull/2847

### CODE REVIEW

File: _data/objectfeatures.json [hunk 1]
```diff
@@ -1389,5 +1389,5 @@
    }
   ]
  },
- "ilastik.__version__": "1.4.0rc6"
+ "ilastik.__version__": "1.4.1b16.dev1+ged23825fb.d20240319"
```
Comment (btbest) [inline _data/objectfeatures.json:1392]: Oops?

I guess this isn't really relevant for anything though

**Review by btbest — APPROVED** [review]

File: _data/objectfeatures.json (hunk 1 above)
Comment (k-dominik) [reply _data/objectfeatures.json:1392]: well it's the version of the code I ran it on...

Comment (k-dominik) [reply _data/objectfeatures.json:1392]: I can rerun once the ilastik PR is merged, then the reference should point to something more reachable (but still dev-y)

Comment (btbest) [reply _data/objectfeatures.json:1392]: Sure, though I don't think it's an issue anyway

**Pull request merged**